index.js

```diff
@@ -24,13 +25,22 @@ let LOGGER;
 
 function connect(opts) {
   let stream;
+
   const endpoint = `${opts.region}${opts.server}`;
 
+  dns.lookup(endpoint, (err, address, family) => {
```

Comment:
do you realise that this is asynchronous and as such this hapens in parallel with connect/createConnection? not sure if pulling in a dependency purely for a log message is the right way... 😅

Reply:
yup, why would it being async be bad?
just means we'll sometimes get the log message after we log out connecting

> not sure if pulling in a dependency purely for a log message is the right way...

What's the alternative? we're constantly in the dark about DNS issues, tls.connect doesn't have any useful callbacks or error messages :(

Reply:
out of the scope of this library tbh, customer is in charge of having a working dns service. but was just making sure you were aware that this is async and thus won't serve as a precheck.

Reply:
yup yup, thought about making it sync and throwing an error if we can't DNS but thought it's a bit extreme, especially since reconnecting is normal and it'll force the container to restart